src/asciidoc.ts

```diff
@@ -9,6 +9,19 @@ const TOP_HEADER = /^={1,3} (.*)$/;
 export function extractAsciidocSamples(text: string, p: Processor) {
   const lines = text.split('\n');
 
+  function extractCodeSample(i: number, until: string) {
+    i += 1;
+    const startLine = i;
+    p.setLineNum(startLine);
+
+    for (; lines[i] !== until; i++);
```

Comment:
I realize this is an existing issue, but this will go into an infinite loop if the code sample is improperly closed.

Reply:
Haha I hit it for a bit while making this change... will fix!